api/core/tools/provider/builtin/aws/tools/sagemaker_tts.py

```diff
@@ -21,7 +21,7 @@ class SageMakerTTSTool(BuiltinTool):
     s3_client: Any = None
     comprehend_client: Any = None
 
-    def _detect_lang_code(self, content: str, map_dict: dict = None):
+    def _detect_lang_code(self, content: str, map_dict: Optional[dict] = None):
```

Comment:
This parameter is unused, you can remove it.

Reply:
(ditto) Will remove the unused parameter in the follow-up PR, covering the method's signature and usages.